apps/condo/domains/resident/schema/RegisterServiceConsumerService.js

```diff
@@ -96,9 +98,22 @@ const RegisterServiceConsumerService = new GQLCustomSchema('RegisterServiceConsu
                     organization: { connect: { id: organization.id } },
                 }
 
+                const optionalAttrs = {
+                    ...paymentFeatureAttrs,
+                }
+
+                // User can request explicit validation of some fields. This is required for iOS and Android mobile clients
+                if (Array.isArray(validations)) {
```

Comment:
Why it’s optional? Can we just use it by default?

Reply:
I don't want to supply an empty array as a validations argument, if I don't want any oprtional validations to be run, thus why it is not required